package.json

```diff
@@ -104,6 +104,7 @@
     "terser-webpack-plugin": "^3.0.1",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
Can't we just use the plugin? We effectively have 2 things doing the same.

Reply:
Yes, I will change that in sperate PR.